oem/ibm/libpldmresponder/oem_ibm_handler.hpp

```diff
@@ -115,8 +115,11 @@ class Handler : public oem_platform::Handler
                         disableWatchDogTimer();
                         pldm::responder::utils::clearLicenseStatus();
                     }
-                    else if (propVal ==
-                             "xyz.openbmc_project.State.Host.HostState.Running")
+                    else if (
+                        (propVal ==
+                         "xyz.openbmc_project.State.Host.HostState.Running") ||
+                        (propVal ==
+                         "xyz.openbmc_project.State.Host.HostState.Standby"))
```

Comment:
Does it make any difference to surveillance? Would like Sagar to comment and test once before this gets merged